cdb2api/cdb2api.c

```diff
@@ -2632,6 +2611,7 @@ static int cdb2_next_record_int(cdb2_hndl_tp *hndl, int shouldretry)
     int len;
     int rc;
     int num_retry = 0;
+    debugprint("entering\n");
```

Comment:
We already have way too many debug statements in next_record. Let's not add more unless we really really need it.

Reply:
You are right, we have a good number of debug stmts there -- i added more because i was debugging an issue. I will remove them because not necessary generally and one can add later if needed. 
thanks.